src/overlays/actors/ovl_En_Encount2/z_en_encount2.h

```diff
@@ -35,6 +35,4 @@ typedef struct EnEncount2 {
 
 #define GET_ENCOUNT2_SWITCH_FLAG(actor) ((s16)(((Actor*)actor)->params & 0x7F))
```

Comment:
Here too.

Reply:
Don't worry about the params macros. They should all be cleaned up in #992.